modules/core/test/src-js/smithy4s/Platform.scala

```diff
@@ -0,0 +1,6 @@
+package smithy4s
+
+object Platform {
```

Comment:
minor in this instance: I like when platform or scala version specific implementation like that implement a common interface defined in the global source folder

that way when you work on the specific implementation, you know what you need to implement

Reply:
I tend to agree generally speaking, but, honestly, in this case I'm not bothered.